### DISCUSSION TITLE
Acessar no navegador minha página php

### DISCUSSION THREAD

**Discussion by Daniel-Ralvs** [Daniel-Ralvs/Grafhy#3] [Q&A] [answered]
Conectei o banco de dados e preciso saber se deu certo, como abrir o arquivo no navegador?

**Comment by Daniel-Alvarenga** [comment, marked as answer]
você precisa deixar a página no htdocs dentro da pasta do Xampp e ir no navegador, digitar o nome da conexão em que o está, passar o nome da porta e o caminho para seu arquivo php como se ele estivesse passando o caminho a partir da htdocs exemplo: localhost:80/meu_site/meu_arquivo.php
